tun/tun.go

```diff
@@ -18,12 +18,36 @@ const (
 )
 
 type Device interface {
-	File() *os.File                 // returns the file descriptor of the device
-	Read([]byte, int) (int, error)  // read a packet from the device (without any additional headers)
-	Write([]byte, int) (int, error) // writes a packet to the device (without any additional headers)
-	Flush() error                   // flush all previous writes to the device
```

Comment:
I guess this `Flush()` was a no-op everywhere anyway... This is a relic from the early days of Wintun when the API used WriteFile!